Projects/Chat/Sources/Views/ChatScreen.swift

```diff
@@ -20,8 +20,10 @@ struct ChatScreen: View {
             infoCard
                 .padding(.bottom, -8)
             ChatInputView(vm: vm.chatInputVm)
+                .padding(.bottom, 32)
         }
         .dismissKeyboard()
+        .edgesIgnoringSafeArea(.bottom)
```

Comment:
.edgesIgnoringSafeArea(.bottom)
When we add this, view doesn't move together with the keyboard 😢 
I think its not needed